Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -26,6 +28,55 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public class PlayModeTestUtilities
     {
+        const string playModeTestSceneName = "PlayModeTestScene";
```

Comment:
Is it possible that this name could already be loaded by SceneManager? Maybe something a bit more unique like "Microsoft.MixedReality.Toolkit.Tests.PlayModeTestScene", or "MixedRealityToolkit.PlayModeTestScene"? or, suffix it with the current date?